Lib/multiprocessing/popen_spawn_win32.py

```diff
@@ -68,7 +68,7 @@ def __init__(self, process_obj):
         else:
             env = None
 
-        with open(wfd, 'wb', closefd=True) as to_child:
+        with open(wfd, 'wb', closefd=False) as to_child:
```

Comment:
Why is that? Is it a leftover from previous attempts?

Reply:
Good catch, sorry about that.